src/Akka.Persistence.MongoDb/Journal/MongoDbJournal.cs

```diff
@@ -34,13 +35,12 @@ public class MongoDbJournal : AsyncWriteJournal
         private Lazy<IMongoDatabase> _mongoDatabase;
         private Lazy<IMongoCollection<JournalEntry>> _journalCollection;
         private Lazy<IMongoCollection<MetadataEntry>> _metadataCollection;
-
+        public static readonly AtomicCounter Counter = new AtomicCounter(0);
```

Comment:
Need to be removed

Reply:
Don't put testing infrastructure onto the journal itself, and definitely not in the public API. A separate `static class` with the `Counter` property inside the test suite is a more suitable location.